composer.json

```diff
@@ -14,14 +14,14 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^5.6 || ^7.0",
```

Comment:
@yukoff your right but we dont want to change php deps inside a minor release. we currently talking about ban 5.5 for 2.0 release.

It should be possible to use both http-cache version 2 and version 1? isnt it?